ctru-rs/src/lib.rs

```diff
@@ -88,18 +87,9 @@ fn panic_hook_setup() {
 
         // Only for panics in the main thread
         if main_thread == std::thread::current().id() && console::Console::exists() {
-            println!("\nPress SELECT to exit the software");
+            println!("\nThe software will exit in 5 seconds");
 
-            match Hid::new() {
-                Ok(mut hid) => loop {
-                    hid.scan_input();
-                    let keys = hid.keys_down();
-                    if keys.contains(KeyPad::SELECT) {
-                        break;
-                    }
-                },
-                Err(e) => println!("Error while intializing Hid controller during panic: {e}"),
-            }
+            std::thread::sleep(std::time::Duration::from_secs(5));
```

Comment:
I could see this making panics a little harder to debug sometimes, but it does seem like it's tricky to make sure the Hid / console are initialized to do the other one... I suppose apps can always install their own hook to record panic info somewhere, if they need.

Reply:
Losing this functionality would be quite disastrous, it's one of the things I enjoy the most about our toolchain. I'll look a bit into other possibilities, otherwise I'll push a version without any panic hook.

Reply:
I've decided to remove the panic handler entirely. Let's see where this will take us.